cmake/podioTest.cmake

```diff
@@ -23,7 +23,12 @@ function(CREATE_PODIO_TEST sourcefile additional_libs)
   add_executable( ${name} ${sourcefile} )
   add_test(NAME ${name} COMMAND ${name})
 
-  target_link_libraries(${name} PRIVATE TestDataModel ExtensionDataModel ${additional_libs})
+if (ENABLE_SIO)
+  target_link_libraries(${name} PRIVATE TestDataModel ExtensionDataModel ${additional_libs} TestDataModelSioBlocks ExtensionDataModelSioBlocks)
```

Comment:
I am not sure this is necessary, as the `SioBlocks` libraries are only dynamically loaded during runtime, and I assume that the linker will simply ignore them during linking, because nothing in the actual executable depends on them.

Reply:
> will simply ignore them during linking

Only if the linker flag `--as-needed` is set?